{{cookiecutter.repo_name}}/Makefile

```
@@ -0,0 +1,29 @@
+lint:
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)